sampler/stories/default/ColorPicker.js

```diff
@@ -18,6 +18,7 @@ storiesOf('Agate', module)
 			<ColorPicker
 				direction={select('direction', prop.direction, ColorPicker, 'right')}
 				defaultValue={text('defaultValue', ColorPicker, '#3467af')}
+				extended={boolean('extended', ColorPicker)}
```

Comment:
`extended` is managed by the HOC in `ColorPicker` so we don't need the knob here.